src/components/OrderList/index.tsx

```diff
@@ -1,4 +1,4 @@
-import OrderListModel from '../../models/orderListModel';
+import OrderListModel from '../../models/orderModel';
```

Comment:
import OrderModel

Reply:
and it's better to import from models like so `import {OrderModel} from '../../models'`

there will be intellisense